components/brave_wallet/resources/solana_provider.js

```diff
@@ -30,6 +29,7 @@
     createPublickey: {
       value: (base58Str) => {
         console.warn('This API is intended for internal use.')
+        const solanaWeb3 = require('@solana/web3.js')
```

Comment:
Confirming this version hasn't been updated and is still running v1.36.0 which I reviewed in PR #12554